### PR TITLE
add flags force all spaces have only one partition

### PR DESCRIPTION
**Pull request by knightXun**
rt, this is for cloud

### CODE REVIEW

File: src/meta/processors/partsMan/CreateSpaceProcessor.cpp [hunk 1]
```diff
@@ -102,6 +103,10 @@ void CreateSpaceProcessor::process(const cpp2::CreateSpaceReq& req) {
         return;
     }
 
+    if ( force_single_part ) {
```
Comment (yixinglu) [inline src/meta/processors/partsMan/CreateSpaceProcessor.cpp:106]: `FLAGS_force_single_part`

Comment (knightXun) [reply src/meta/processors/partsMan/CreateSpaceProcessor.cpp:106]: good point

**Review by CPWstatic — COMMENTED** [review]
Would it be better put this global config in graph?

File: src/meta/processors/partsMan/CreateSpaceProcessor.cpp [hunk 2]
```diff
@@ -102,6 +103,10 @@ void CreateSpaceProcessor::process(const cpp2::CreateSpaceReq& req) {
         return;
     }
 
+    if (FLAGS_force_single_part) {
```
Comment (Shylock-Hg) [inline src/meta/processors/partsMan/CreateSpaceProcessor.cpp:106]: I think you can set the FLAGS_max_parts_num to 1 instead of add a new flags.

Comment (knightXun) [reply src/meta/processors/partsMan/CreateSpaceProcessor.cpp:106]: No, FLAGS_max_parts_num is for total space partitions, not for single space partition

Comment (Shylock-Hg) [reply src/meta/processors/partsMan/CreateSpaceProcessor.cpp:106]: Ok.

**Pull request closed without merging**